packages/generator-emakinacee-react/generators/component/index.js

```diff
@@ -42,14 +42,24 @@ class ComponentGenerator extends Generator {
     _getPathOptions() {
         const pathOptions = {};
         const split = this.options.path.split('/');
+        let relPath = '../../';
```

Comment:
This "magic value" could be a const at the top level.

Reply:
Mh 🤔 to me it would look strange to write something like:

```javascript
const relativePath = '../../':
...
let relPath = relativePath;
```

Would it be less magical to you to just call it `relativePath` or `relativePathToStylesheet`?

Or did I get your hole comment wrong?

Reply:
Maybe the code got changed, but my impression was that the value would not change. So this is comment does not apply anymore.